ietf-scitt-charter.md

```diff
@@ -4,25 +4,32 @@ Over the years, rapid technological advancements have motivated organizations to
 While these improvements help organizations increase efficiency and swiftly bring innovations to market, the rapid increase in scale, size, and complexity of supply chains has led to more frequent and sophisticated supply chain attacks.
 The traditional methods of safeguarding supply chains (e.g., pre- and post-audit methodologies) are no longer adequate.
 
+The output of the SCITT WG is a set of standards that define the essential building blocks enabling the security of supply chain systems and assisting implementers in securing deployments.
+For example, a public computer interface system could report its software composition, which can be compared against known software compositions for such a device, as recorded in a public append-only transparent registry.
+Therefore, providing an individual using the system with confidence that it will behave as and when expected, consistently and without deviation.
+
 Problem Statement
 =================
 It is challenging to manage the compliance of products across end-to-end, global supply chains. 
 
 Some of the fundamental security issues that face the supply chain ecosystem today are as follows:
 
-1. A single product is composed of multiple sub-products coming from different suppliers. There is no agreed-upon standard to compose information from different producers.
+1. A single product is composed of multiple sub-products coming from different suppliers. There is no agreed-upon standard to compose information from different producers as every vendor is doing it differently.
 2. There are no APIs defined for automated publishing, retrieval, or independent verification of the information above.
 3. The absence of decentralized, globally interoperable, transparent services to publish the supply chain data.
 4. The lack of sufficient standards for independently verifying the presence of supply chain data in tamper-proof data stores.
+5. Software consumers have no trustworthy way to verify that a software signature on a software package is legitimate.
```

Comment:
Since there are many folks out there who will tell you, my PGP verification system works just fine and is plenty trustworthy, perhaps we could focus instead on closing the gap from a methodology perspective of how the attestations generated are consumed in a uniform way across environments to allow for easy adoption by any existing ecosystem whose current status may fall under bullet 5

```suggestion
5. Fractured verification methodologies across software distribution ecosystems create inconsistent security guarantees for end users.
```

Reply:
(5) sounds related to @rjb4standards's comment made here: https://github.com/ietf-scitt/charter/pull/18/files#r957557301

Reply:
It seems we could preserve the existing 5 and add the proposed new 5 as 6 as they both provide justification for the purpose
+6. Fractured verification methodologies across software distribution ecosystems create inconsistent security guarantees for end users.

Reply:
As the "old" (5.) is a verbatim quote from Dick at the BoF and he is okay with the change, I am okay with the change. Was about to merge, but will give folks 12h to object.

Reply:
Henk, my suggestion is to keep the old 5 and add John's recommendation as 6

Reply:
What I am reading is that old (5.) and new (6.) do not read too redundant? I am neutral on the issue. Any objections to simply adding (6.)?

Reply:
No they are not independent statements, but can be clubbed using OR

Reply:
So it should be joined using OR, either lacking verifying methods or fragmented verifying methods

Reply:
@yogeshbdeshpande they are certainly all within the same context of verification, but the new 6 is more abstract. Using an ice cream analogy, 6 is like saying "ice cream is made from dairy products and other ingredients", which is also true for a more direct, less abstract statement, like the current 5,  i.e. Chocolate ice cream is made from whole milk, sugar and chocolate flavoring, which the supplier of the chocolate flavoring cannot be determined or verified.

Reply:
ok, in that case, we should really swap (5) and (6).
Reason been current (6) talks about SW distribution systems while current (6) talks about end user!

Reply:
No objections to swapping 5 and 6, so that John's proposal becomes 5 and the original 5 becomes 6.